README.md

```diff
@@ -1,41 +1,196 @@
-/group_vars : Create location folder
-/host_vars : Create host folder for every openwrt device
-Done!
 
+##### owm.yml
+
+This file defines some values to get the location to the [OpenWifiMap](https://openwifimap.net).
+
+```sh
+---
+location_nice: roof top, Example-Street 24, 10573 Berlin        # any string that describes your location. Could be an address or something
+latitude: 52.484948320
+longitude: 13.443380903
+```
+
+##### networks.yml
+
+This file is the most important one of your setup. It defines IP-Addresses, WIFI-Properties, Mesh and so on. Lets have a (shortened) example of a `networks.yml` from the magda-location:
+
+```sh
+---
+ipv6_prefix: "2001:bf7:860::/56"
+
+# Mesh: 10.31.83.60/30
+networks:
+  - vid: 10                   # vlan-id
+    role: mesh                # what this vlan does (mesh vs. dhcp)
+    name: mesh_sama
+    prefix: 10.31.83.60/32    # single ipv4-address for meshing
+    ipv6_subprefix: -1        # take an address from the back of the IPv6-block
+
+  - vid: 11
+    role: mesh
+    name: mesh_ost
+    prefix: 10.31.83.61/32
+    ipv6_subprefix: -2
+
+  - vid: 42
+    role: mgmt                # create a management vlan in which we can reach every device on this site for maintenance
+    prefix: 10.31.83.112/28
+    gateway: 1
+    dns: 1
+    ipv6_subprefix: 1
+    assignments:              # assign static(!) addresses from mngt-network to individual devices/interfaces.
+      magda-core: 1           # core router gets 1st address. In result it will be reachable at 10.31.83.113
+      magda-switch: 2         # 10.31.83.114 ...
+      magda-sama: 3           # interface for mesh link to sama def'd at vid-10 (see above) gets 3rd address (for mngt only)
+      magda-ost-5ghz: 4       # mesh link on vid-11
+      magda-ap1: 5
+      magda-ap2: 6
+      magda-ap3: 7
+      magda-ap4: 8
+
+  - vid: 40
+    role: dhcp                # dhcp server will run on core-router and serve it's network on vid 40
+    prefix: 10.31.83.192/26
+    ipv6_subprefix: 0
+    assignments:              # assign static addresses to devices
+      magda-core: 1
+
+location__channel_assignments_11a_standard__to_merge:
+  # AP-id, wifi-channel, bandwidth, txpower
+  magda-ap1: 36-20-15
+  magda-ap2: 40-20-15
+  magda-ap3: 44-20-15
+```
+
+### host_vars/
+
+The `host-vars`-dir contains one directory for every OpenWrt-device. The directory-name is the routers hostname. Currently, in that directory theres only one file, called `base.yml`. One Example for that file:
+
+```sh
+---
+
+location: sama                  # locations short name, like def'd in location-directories
+role: corerouter                # devices role. Could either be 'corerouter', 'ap' or 'uplink_gateway'
+target: "ipq40xx/generic"       # target of router model
+model: "avm_fritzbox-7530"      # model name like written in the corresponding file name in group_vars/
+
+wireless_profile: freifunk_default      # activates wifi with freifunk-default-settings on this device
```

Comment:
.. by overriding default wireless profile for corerouters, which is the profile `disable`

Reply:
added in new commit-version